src/main/java/net/runelite/data/dump/MediaWikiTemplate.java

```diff
@@ -190,6 +191,34 @@ public static MediaWikiTemplate parseLua(final String data)
 		return new MediaWikiTemplate(out);
 	}
 
+	/**
+	 * Looks for and parses the `Switch infobox` into a {@link MediaWikiTemplate} and then iterates over the `item#` values.
+	 * Attempts to parse each `item#` value via `parseWikiText`, matching the `name` attribute. null values are ignored
+	 *
+	 * @param name only parses MediaWikiTemplates from `Switch infobox` if matches this value. (case insensitive)
+	 * @param baseTemplate the {@link MediaWikiTemplate} representation of the `Switch infobox` to parse from
+	 * @return List of all valid {@link MediaWikiTemplate}s matching `name` from `baseTemplate`s `item#` values
+	 */
+	public static List<MediaWikiTemplate> parseSwitchInfoboxItems(final String name, final MediaWikiTemplate baseTemplate)
```

Comment:
This can be just part of dumper that needs it?

Reply:
Alright, i wasn't sure if you would need this for items or something in the future, I'll close this PR and update the other one.